notebooks/example_notebook_error_labeling.ipynb

```diff
@@ -0,0 +1,240 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib\n",
+    "matplotlib.use('Agg')\n",
+    "from responsibleai_vision.common.constants import ImageColumns\n",
```

Comment:
seems strange to use responsibleai_vision here - perhaps just use the string constants to avoid this import

Reply:
maybe we eventually should add notebook testing in this repo - but it's ok to merge as is for now

Reply:
Fixed!